tools/setup_pyenv.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+sudo apt-get update
+sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev \
+libreadline-dev libsqlite3-dev wget curl llvm libncursesw5-dev xz-utils \
+tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev clang
+
+curl http://pyenv.run | bash
+
+echo 'export PYENV_ROOT="$HOME/.pyenv"
+command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"
```

Comment:
`$PYENV_ROOT` is undefined at this point

Reply:
even with the export in the previous line?

Reply:
you're right, I lost count of the quotes